telosnetwork/telos-wallet
# Adding prices to WTLOS and STLOS tokens when no indexer

# Fixes #524 

## Description
In the case of the indexer not being healthy, we need a fallback to show at least the price for the system tokens.

## Test scenarios
- go to [this link](https://deploy-preview-537--wallet-staging.netlify.app/)
- login with any method
  - you should see the price for the system tokens (TLOS, WTLOS & STLOS)

## Screenshots
![image](https://github.com/telosnetwork/telos-wallet/assets/4420760/25e7cd56-279a-482f-9816-626271d069e8)

## Checklist:
